samples/daily-prebuilt/permissions-can-admin/src/pages/api/rooms.js

```diff
@@ -1,28 +1,30 @@
 export default async function handler(req, res) {
+  // This handler only accepts POST requests.
+  const { method } = req;
+  if (method !== 'POST') {
+    res.status(405).end(`${method} Not Allowed`);
+    return;
+  }
+
   try {
-    switch (req.method) {
-      case 'POST':
-        const options = JSON.parse(req.body);
-        // Make request to Daily's REST API /rooms endpoint
-        // https://docs.daily.co/reference/rest-api/rooms/create-room
-        const dailyRoomRes = await fetch('https://api.daily.co/v1/rooms/', {
-          method: 'POST',
-          headers: {
-            'Content-Type': 'application/json',
-            Authorization: `Bearer ${process.env.DAILY_API_KEY}`,
-          },
-          body: JSON.stringify(options),
-        });
-        const room = await dailyRoomRes.json();
-        res.status(200).json({ room });
-        break;
-      default:
-        res.status(405).end(`${req.method} Not Allowed`);
-        break;
-    }
+    const options = JSON.parse(req.body);
+    // Make request to Daily's REST API /rooms endpoint
+    // https://docs.daily.co/reference/rest-api/rooms/create-room
+    const dailyRoomRes = await fetch('https://api.daily.co/v1/rooms/', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${process.env.DAILY_API_KEY}`,
+      },
+      body: JSON.stringify(options),
+    });
+    const room = await dailyRoomRes.json();
+    res.status(200).json({ room });
   } catch (error) {
+    const msg = 'Error while creating room';
+    console.error(`${msg}: ${JSON.stringify(error)}`);
     res.status(500).json({
-      error,
+      error: `${msg}; check server logs`,
```

Comment:
This is _not_ related to eslint, but usually for server-side logic best practice is not to return the entire error to the client because there may be sensitive information in it that clients should not have access to. Ofc this being a demo that's unlikely, _but_ I figure we may as well follow best practice for the sake of illustration. So my suggested approach is to _log_ the full error and then send something a little more generic to the client.